### PR TITLE
Datatables styling v3

### PR DESCRIPTION
**Pull request by SueJStevens**
Moved Datatables variable to top of logic.js so that it can initialize in global scope. 
Additional styling changes.

### CODE REVIEW

File: assets/javascript/logic.js [hunk 1]
```diff
@@ -31,27 +36,27 @@ $(function () {
    *  For each company, query IEX for stock information and query Stocktwits 
    *  for trending tweets.
    */
-  database.ref().once("value").then(function(snapshot){
+  database.ref().once("value").then(function (snapshot) {
 
-    if(!snapshot.val()){
+    if (!snapshot.val()) {
```
Comment (narasimhan-ramesh5) [inline assets/javascript/logic.js:41]: Did you add these additional spaces ? Or did they get added automatically? Either way I'm ok with it.

File: assets/javascript/autocomplete.js [hunk 2]
```diff
@@ -183,5 +183,4 @@ $(function () {
 
 
 
-}); //end on ready function
-
+}); //end on ready function
```
Comment (narasimhan-ramesh5) [inline assets/javascript/autocomplete.js:186]: On GitHub I see a warning because there isn't a newline at the end of line.

Comment (SueJStevens) [reply assets/javascript/autocomplete.js:186]: Thanks, I'm trying to make sure those are removed from our project, but it doesn't really matter.

File: assets/javascript/logic.js (hunk 1 above)
Comment (SueJStevens) [reply assets/javascript/logic.js:41]: I did add the spaces, but it was automatic.  Right-click, format document.  We can work on making sure our VS is set up to match in the future so we aren't adding or subtracting spaces.

**Review by narasimhan-ramesh5 — APPROVED** [review]

**Pull request merged**